docs/development_guide/openshift/openshift_nightly.md

```diff
@@ -70,3 +70,83 @@ OpenShift 4 requires that any container programming iptables has privileged esca
+
+### Upgrading the OpenShift Cluster
+
+As OSM now explicitly lists a kubeVersion in the chart.yaml, the minimum Kubernetes version that OSM is compatible with (and for which Helm will block installations for lower versions), the OpenShift cluster will need to be periodically updated so that the Kubernetes version reflects the kubeVersion of the chart currently in main (for tag `latest-main`). The Kubernetes version that corresponds to each OpenShift Container Platform release can be found on the (release notes)[https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html] on the OpenShift docs site. For instance, to update to Kubernetes version v1.23.x, we will need to update the OpenShift cluster to 4.10.x or above.
+
+#### Setting the upgrade channel using the OC cli
+
+OpenShift's "upgrade channels" allow users to perform minor version updates for an OpenShift cluster. For instance, to upgdate the cluster from 4.7 to 4.8, the user would need to currently be on one of the 4.8 upgrade channels: `stable-4.8`, `fast-4.8`, `candidate-4.8`, or `eus-4.y` (only for even numbered 4.y cluster releases, ex: 4.8). For upgrading the cluster used for the OpenShift nightly job, we should stick with the `stable-x.y` channels. 
+
+Run the following command to switch to a particular upgrade channel:
+
+```bash
+oc patch clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<stable-x.y>"}]'
+```
+
+You should see this message in the output: 
+
+```console
+clusterversion.config.openshift.io/version patched
+```
+
+Run the following command to ensure that the upgrade channel has been set properly:
+
+```bash
+oc get clusterversion -o json|jq ".items[0].spec"
+```
+
+The output should have the correct channel and new version to update to:
+
+```json
+{
+  "channel": "stable-x.y",
+  "desiredUpdate": {
+    "version": "x.y.z"
+  }
+}
+```
+
+More information about [upgrade channels](https://docs.openshift.com/container-platform/4.10/updating/understanding-upgrade-channels-release.html) can be found on the OpenShift docs site. 
+
+#### Upgrading using the OC cli
+
+Now that you've set the upgrade channel, view the available updates with the following command: 
+
+```bash
+oc adm upgrade
+```
+
+The output should accurately reflect the current OpenShift cluster version, as well as the new version that you are upgrading to: 
+
+```console
+Cluster version is x.<y-1>.w
+
+Updates:
+
+VERSION IMAGE
+x.y.z   <image>
+```
+
+Once you have verified that the version is correct, you can upgrade the cluster with one of the two following commands: 
+
+```bash
+# Update to the latest version
+oc adm upgrade --to-latest=true
+```
+
+```bash
+# Update to a specific version
+oc adm upgrade --to=<version>
+```
+
+Once you run the upgrade command, you should see the following output as a confirmation that the upgrade is starting:
```

Comment:
Is the Openshift cluster long lived or do we create it each time we run the e2e suite?

Reply:
The cluster is long-lived, we log in to that cluster using OC cli for the nightly job and run the e2e suite.